bin/oneoff/deprecate_unused_poste_urls.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require_relative './poste_urls_constants'
+
+# This script is a dry-run unless we change DRY_RUN to false
+DRY_RUN = true
+
+if DRY_RUN
+  puts "DRY RUN..."
+else
+  puts "Starting update..."
+end
+
+DB.transaction do
+  # mark all poste_urls that are not in urls_to_keep as deleted_at = now
+  # this will take a long time to run as it iterates over the entire poste_urls table
+  urls_kept = []
+  deleted_at_time = DateTime.now
+  DB[:poste_urls].each do |poste_url|
+    if !POSTE_URLS_TO_KEEP.include?(poste_url[:id])
+      begin
+        DB[:poste_urls].where(id: poste_url[:id]).update(deleted_at: deleted_at_time)
+      rescue => e
```

Comment:
Rescue everything?

Reply:
yes, we want to try to update everything and handle any errors later since we don't need to do the update all at once.